mender-update/update_module/v3/update_module.hpp

```diff
@@ -72,8 +81,62 @@ class UpdateModule {
 	Error Cleanup();
 
 private:
+	Error PrepareStreamNextPipe();
+	Error OpenStreamNextPipe(ExpectedWriterHandler open_handler);
+	Error PrepareAndOpenStreamPipe(const string &path, ExpectedWriterHandler open_handler);
+	Error PrepareDownloadDirectory(const string &path);
+	Error DeleteStreamsFiles();
+
+	void StartDownloadProcess();
+
+	void StreamNextOpenHandler(io::ExpectedAsyncWriterPtr writer);
+	void StreamOpenHandler(io::ExpectedAsyncWriterPtr writer);
+
+	void StreamNextWriteHandler(size_t expected_n, size_t written_n, error::Error err);
```

Comment:
Nit: Above you're just using `Error`, while here you namespace it

Reply:
Indeed. The plain `Error` is only used in this file, and I think I prefer the namespaced one for clarity (it's also what we use practically everywhere). So I reverted to namespaced.